tests/integration/deleted_with/nodejs/failsOnDelete.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright 2016-2023, Pulumi Corporation.  All rights reserved.
```

Comment:
It is very lame we have to write each of these integration test 6 times (python, nodejs, go, dotnet, java, yaml). Are we ever going to prioritize matrix testing to fix this?

Reply:
Yes, we should talk more about how/when we can get to it.